src/app/component/ProductGallery/ProductGallery.style.scss

```diff
@@ -48,9 +48,30 @@
         }
     }
 
+    .react-transform-component,
+    .react-transform-element,
+    &-SliderImage {
+        height: 100%;
+        width: 100%;
+    }
+
     &-SliderImage {
         &:not(.ProductGallery-SliderImage_isPlaceholder) {
             background: none;
```

Comment:
This one has to be removed, as if you try opening the page separately (in new tab). It will show white space instead of image.